UN-OCHA/unocha-site
# [UNO-780] Title and text fields for the donors paragraph type

Refs: UNO-780

Replaces https://github.com/UN-OCHA/unocha-site/pull/339

This adds title and text fields to the donors paragraph type.

## Tests

1. Checkout the branch, clear the cache, import the config
2. Go to `/our-founding` and edit it, replace the title and text by what is in the ticket
3. Check that the title and text appear in the form
4. Save and check that the title and text appear on the page
5. Edit the `/Afghanistan` page for example 
6. Check that the donors paragraphs have a `Top donors (FTS)` type of title (label of the figure and provider in parentheses)
7. Edit a donors paragraph, add a title and text, save and check that it's displayed instead of the default title
